README.rst

```diff
@@ -72,3 +67,219 @@ response, e.g.
+        # https://docs.wagtail.io/en/stable/advanced_topics/deploying.html#cloud-storage
+        # Not having this setting may have consequences in losing files.
+        AWS_S3_FILE_OVERWRITE = False
+
+        # Default ACL for new files should be "private" - not accessible to the
+        # public. Images should be made available to public via the bucket policy,
+        # where the documents should use wagtail-storages.
+        AWS_DEFAULT_ACL = "private"
+
+        # We generally use this setting in the production to put the S3 bucket
+        # behind a CDN using a custom domain, e.g. media.llamasavers.com.
+        # https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#cloudfront
+        if "AWS_S3_CUSTOM_DOMAIN" in os.environ:
+            AWS_S3_CUSTOM_DOMAIN = os.environ["AWS_S3_CUSTOM_DOMAIN"]
+
+        # When signing URLs is facilitated, the region must be set, because the
+        # global S3 endpoint does not seem to support that. Set this only if
+        # necessary.
+        if "AWS_S3_REGION_NAME" in os.environ:
+            AWS_S3_REGION_NAME = os.environ["AWS_S3_REGION_NAME"]
+
+        # This settings lets you force using http or https protocol when generating
+        # the URLs to the files. Set https as default.
+        # https://github.com/jschneier/django-storages/blob/10d1929de5e0318dbd63d715db4bebc9a42257b5/storages/backends/s3boto3.py#L217
+        AWS_S3_URL_PROTOCOL = os.environ.get("AWS_S3_URL_PROTOCOL", "https:")
+
+You can set the following environment variables to configure your bucket if you
+used the above snippet.
+
+* ``AWS_STORAGE_BUCKET_NAME`` - set to ``media.llamasavers.com``.
+* ``AWS_S3_CUSTOM_DOMAIN`` - set to ``media.llamasavers.com``.
+* ``AWS_S3_REGION_NAME`` - set to your AWS region name, e.g. ``eu-west-2``.
+
+You can use one of the methods to provide `boto3 with credentials`__. E.g. to
+use environment variables, you need to set the following variables:
+
+* ``AWS_ACCESS_KEY_ID``
+* ``AWS_SECRET_ACCESS_KEY``
+
+.. _Boto3Credentials: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html
+
+__ Boto3Credentials_
+
+Now the storage should be configured and working, e.g. editors should be able to
+upload images and documents in Wagtail admin.
+
+Set up ``wagtail-storages``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Install ``wagtail-storages`` itself.
+
+.. code:: sh
+
+   pip install wagtail-storages
+
+
+Add ``wagtail_storages`` to your ``INSTALLED_APPS`` in your settings file.
+
+.. code:: python
+
+   # settings.py
+
+   INSTALLED_APPS = [
+       # ... Other apps
+       "wagtail_storages",
+       # ... Other apps
+   ]
+
+With having that set up that, ACLs should be updated if documents are moved to
+private collections.
+
+Set up front-end cache invalidation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If edge cache is set up on the custom domain, ``media.llamasavers.com``, the CDN
```

Comment:
brackets rather than commas preserves flow

> custom domain, ``media.llamasavers.com``, the CDN
→
custom domain (``media.llamasavers.com``), the CDN